R/tutorial.R

```diff
@@ -0,0 +1,44 @@
+#' Create a tutorial.
+#'
+#' Creates new tutorial in `inst/tutorials`. Tutorials are interactive R Markdown documents built with the `learnr` package.
```

Comment:
Can you please make sure this line is wrapped to ~80 characters? (I think it's a bit long currently)